04-rstudio-tour.Rmd

```diff
@@ -1,38 +1,277 @@
+
+</details>
+
+### Updating RStudio
+
+It is also a good idea to keep RStudio up-to-date. New features become available as the Posit team works on developing RStudio. So if you already have RStudio, you might want to check to see if your version is up-to-date.
+
+To check for updates you can go to the `Help` menu at the top of RStudio and then click on `Check for Updates`. 
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Check for updates for RStudio in the Help menu of RStudio."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_50")
+```
+
+## Navigating RStudio
+
+Now that you hopefully have RStudio running on your machine, we will walk you through some of the major features that can really help you with your data analyses.
+
+### Default Layout
+
+First it is important to be familiar with the layout. When you first open RStudio you will see 3 panes.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "When RStudio is first opened you will see 3 panes"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_67")
+```
+
+<details><summary>If your RStudio looks different click here.</summary>
+
+Click on the top menu of your RStudio - click where it says `Edit` --> `Preferences` --> `Pane Layout`.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Select Edit and then Preferences to get to the menu about pane layout."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g22370a5f292_430_0")
+```
+
+The Pane Layout menu enables you to change the layout. The image below shows the default settings. 
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "The Pane Layout menu allows you to modify the layout of RStudio. This shows the default settings with Source on the top left, Console on the bottom left, Environment, History, Connections, and Build on the top right, and Files, Plots, Packages, Help, and VCS on the bottom right."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g22370a5f292_430_5")
+```
+
+</details>
+
+
+The pane on the left (labeled "Pane 1" in the image) is where we write code.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "The pane on the left is where we write and test code."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_81")
+```
+
+The pane on the top right (labeled "Pane 2" in the image) is where we can see what objects we have created and are actively in memory (meaning they can be used at that time) in what is called the "Environment".
```

Comment:
```suggestion

Depending on what code we write in the Console tab, something will be echoed back to us or R will otherwise save it to its 'Environment'. The Environment in R can be thought of as R's working memory.

To play with the Console tab, let's practice using the assignment operator `<-`. The assignment operator is what creates R objects that are stored in the Environment. Aka the `<-` tells R to remember something for us. 

Try this bit of code: 
```
one_to_ten <- 1:10
```
If you look in the pane on the top right (labeled "Pane 2" in the image) we can see what objects we have created (and you should see `one_to_ten` there. The objects listed here are actively in R's memory or Environment. If we attempt to reference an object that is not in the Environment, R will give us an error. For example, if in the Console window we type `dog` R will say `Error: object 'dog' not found` because we have not created an object called `dog` yet.

The Console window is great for testing things out but note that for our code to be reproducible, we will need to eventually have it written down in a script or notebook because the Console window won't be something we can return to easily to rerun our code.

```

Reply:
Did you see the later example for this? I kinda just wanted this first part to orient people and then give them these examples, which is what I did next.

Reply:
I don't want to overwhelm them when they are first getting oriented

Reply:
Where did we leave off with this bit?